examples/go/go-client/terraform/resources/api/v2010/README.md

```diff
@@ -5,22 +5,22 @@
 
 Name | Type | Requirement | Description
 --- | --- | --- | ---
-**x_twilio_webhook_enabled** | string | Optional | 
-**recording_status_callback** | string | Optional | 
-**recording_status_callback_event** | list(string) | Optional | 
-**sid** | string | *Computed* | 
-**status** | string | Optional | 
-**pause_behavior** | string | Optional | 
+**x_twilio_webhook_enabled** |  | Optional | 
+**recording_status_callback** |  | Optional | 
+**recording_status_callback_event** |  | Optional | 
+**sid** |  | *Computed* | 
+**status** |  | Optional | 
+**pause_behavior** |  | Optional | 
```

Comment:
This doesn't look right.

Reply:
Hmm.. sonarlint said the getFullType method doesn't get used, but I guess it gets used under the hood. I'll add it back in.

Just for my understanding - is this due to the lombok.value annotation?

Reply:
Sonar doesn't know it's being used in the mustache template.

Reply:
Ah didn't know it got used there, that makes sense.